### ISSUE TITLE
App name and location

### ISSUE THREAD

**Issue by whmacken** [kdaust/FeasibilityApp#9]
I think we could move the repo to the FLNRO-Smithers-Research organization.
If we are going to add in other tabs like for forest health then maybe we should have a name for the repo and app that is broader.
I thing By BGC Map App or something

**Comment by kdaust** [comment]
I agree. What should we call the app in the url (i.e. right now it's "feasibility" - ByBGCMapApp seems cumbersome for the URL)
